test/common.js

```diff
@@ -149,7 +149,8 @@ function mergeTestConfig(config) {
     host     : process.env.MYSQL_HOST,
     port     : process.env.MYSQL_PORT,
     user     : process.env.MYSQL_USER,
-    password : process.env.MYSQL_PASSWORD
+    password : process.env.MYSQL_PASSWORD,
+    supportBigNumbers: true
```

Comment:
Can you move this to just be set in the one test that needs it, instead of set in every single test? `common.getTestConnection` accepts an optional first argument to provide test-specific configuration to merge with the common config here.